src/SDKs/CognitiveServices/dataPlane/Language/Language/Language/Microsoft.Azure.CognitiveServices.Language.csproj

```diff
@@ -13,7 +13,8 @@
     <![CDATA[
     Microsoft Cognitive Services Text Analytics SDK.
 
-    Note: This is the last update for this NuGet package. It will be succeeded by the <a href="https://www.nuget.org/packages/Microsoft.Azure.CognitiveServices.Language.TextAnalytics/1.0.1-preview">Microsoft.Azure.CognitiveServices.Language.TextAnalytics</a> package.
+    Note: This is the last update for this NuGet package. It will be succeeded by the Microsoft.Azure.CognitiveServices.Language.TextAnalytics package.
+    https://www.nuget.org/packages/Microsoft.Azure.CognitiveServices.Language.TextAnalytics/1.0.1-preview
```

Comment:
@assafi please do not mention the version in the URL
2 months down the line, you might want to hide that version for some reason and you don't want to give a version specific url, rather just give the package url (without the version number)
https://www.nuget.org/packages/Microsoft.Azure.CognitiveServices.Language.TextAnalytics/